ci.sh

```diff
@@ -26,24 +26,29 @@ adx testsetup
 
 # Run tests with set -e - exit on error
 set -e
-echo "Running ./adx test restricted"
-adx test restricted
-echo "Running dhis2harvester tests"
-adx test dhis2harvester
-echo "Running emailasusername tests"
-adx test emailasusername
-echo "Running file_uploader_ui tests"
-adx test file_uploader_ui
-echo "Running pages tests"
-adx test pages
-echo "Running pdfview tests"
-adx test pdfview
-echo "Running scheming tests"
-adx test scheming
-echo "Running validation tests"
-adx test validation
-echo "Running ytp-request tests"
-adx test ytp-request
-echo "Running unaids tests"
-adx test unaids
 
+
+error(){
```

Comment:
In line 12, after adx setup you can checkout adx_develop to feature branch used by Jenkins.